frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -160,6 +168,7 @@ export const Dashboard = () => {
       state: {
         sortedOrder: event.target.value as SortOrder,
         filterOption: state?.filterOption ? state.filterOption : 'no-filter',
+        tableView: state?.tableView ? state.tableView : defaultView,
```

Comment:
change this line to `tableView: state?.tableView !== undefined ? state.tableView : defaultView,`